keps/993-two-phase-admission/kep.yaml

```diff
@@ -2,7 +2,7 @@ title: Two Stage Admission Process
 kep-number: 993
 authors:
   - "@mwielgus"
-status: draft
+status: implemented
```

Comment:
nit: use `implementable`. implemented is basically reserved for GA features. Until then we are free to extend them: https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/0000-kep-process/README.md#kep-workflow

Reply:
Done